tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -26,9 +26,9 @@ test_spi_apa102: apa102@2 {
 	spi-max-frequency = <0>;
 };
 
-test_spi_rf2xx: rf2xx@4 {
+test_spi_rf2xx: rf2xx@3 {
```

Comment:
Same here - document that the SPI addresses should be sequential.

Reply:
Done